scripts/plots.py

```diff
@@ -0,0 +1,18 @@
+import hvplot.pandas
+
+def create_hvplot(idf, title, x_axis=None, y_axis=None, 
```

Comment:
Can we add some docstring and type hint for the functions? Here is one example: https://github.com/autogluon/autogluon/blob/master/common/src/autogluon/common/utils/deprecated_utils.py
Also, consider adding unit tests if possible and make sense

Reply:
Will skip the other functions